skimage/draw/tests/test_draw_nd.py

```diff
@@ -0,0 +1,7 @@
+from skimage.draw import line_nd
+
+
+def test_empty_line():
```

Comment:
Would be nice to have a test with zero-crossing line. And a note in https://github.com/scikit-image/scikit-image/blob/master/doc/release/release_dev.rst . Otherwise, 👍.